jni/themis_session.c

```diff
@@ -527,6 +527,11 @@ JNIEXPORT jbyteArray JNICALL Java_com_cossacklabs_themis_SecureSession_jniGenera
 
     return output;
 }
+JNIEXPORT jbyteArray JNICALL Java_com_cossacklabs_themis_SecureSession_jniGenerateConntect(JNIEnv* env, 
+                                                                                           jobject thiz)
```

Comment:
Looks like formatting checks are still failing 😞

<details>
<summary>CI failure</summary>

```
check jni/themis_session.c 12G[ERRORS]
clang-format-8 jni/themis_session.c | diff -u jni/themis_session.c - && clang-tidy-8 jni/themis_session.c -- -Iinclude -Isrc -Isrc/wrappers/themis/ -I/usr/local/include -fPIC -DBORINGSSL -DCRYPTO_ENGINE_PATH=boringssl -Ithird_party/boringssl/src/include -O2 -g -fno-omit-frame-pointer -fstack-protector -D_FORTIFY_SOURCE=2 -Werror -Wall -Wextra -Wformat -Wformat-nonliteral -Wformat-security -Wnull-dereference -Wshift-overflow -Wshift-negative-value -Wstrict-overflow -Wswitch -Wvla -Wpointer-arith -Wstrict-prototypes -fvisibility=hidden  -I/usr/lib/jvm/java-11-openjdk-amd64/include -I/usr/lib/jvm/java-11-openjdk-amd64/include/linux 2>/dev/null && touch build/obj/jni/themis_session.c.fmt_check
--- jni/themis_session.c	2022-11-01 12:28:16.036063376 +0000
+++ -	2022-11-01 12:29:12.077085163 +0000
@@ -527,7 +527,7 @@
 
     return output;
 }
-JNIEXPORT jbyteArray JNICALL Java_com_cossacklabs_themis_SecureSession_jniGenerateConntect(JNIEnv* env, 
+JNIEXPORT jbyteArray JNICALL Java_com_cossacklabs_themis_SecureSession_jniGenerateConntect(JNIEnv* env,
                                                                                            jobject thiz)
 {
     return Java_com_cossacklabs_themis_SecureSession_jniGenerateConnect(env, thiz);
make: *** [Makefile:452: build/obj/jni/themis_session.c.fmt_check] Error 1
```

</details>

It seems they are triggered by this stray whitespace which is not immediately obvious in the diff:

```suggestion
JNIEXPORT jbyteArray JNICALL Java_com_cossacklabs_themis_SecureSession_jniGenerateConntect(JNIEnv* env,
                                                                                           jobject thiz)
```